library/build.gradle

```diff
@@ -8,6 +19,7 @@ android {
         minSdkVersion 17
         targetSdkVersion 23
         versionCode 1
+        // NOTE: When updating versionName, be sure to update version in publishing group below too
```

Comment:
could we make this a variable so we only need to update it once? I know gradle's scopes are tricky; if you already tried that and it was weird/tough, we can easily follow a comment's directions!